common/utilities/integration_adapters_logger.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import sys
+import time
+
+AUDIT = 25
+
+
+def _add_audit_level_log():
+    """
+    Adds a new log level to the standard python logging module, this level is the `audit` level and is defined as
+    a high level than INFO but lower than  WARNING
+    """
+    logging.addLevelName(AUDIT, "AUDIT")
+
+    def audit(self, message, *args, **kws):
```

Comment:
Do you actually use this audit method? It looks like all logging is done via the `log(level, ...)` method.

Reply:
Good point, I thought it had to be there in order for the logger to print that log level but it seems you only need the `logging.addLevelName` which I'll just move to the global config method.